bark/examples/merging.py

```diff
@@ -109,5 +110,8 @@ def goal(self, world):
   for step in range(0, 20):
     env.step()
     time.sleep(sim_step_time/sim_real_time_factor)
+    
+  df = env.ExtractTimeSeries()
+  print(df)
```

Comment:
remove print?

Reply:
I guess for an example that's sufficient.